src/test/ui/issue-52533.nll.stderr

```diff
@@ -0,0 +1,17 @@
+warning: not reporting region error due to nll
+  --> $DIR/issue-52533.rs:15:16
+   |
+LL |     foo(|a, b| b)
+   |                ^
+
+error: unsatisfied lifetime constraints
+  --> $DIR/issue-52533.rs:15:16
+   |
+LL |     foo(|a, b| b)
+   |          -  -  ^ closure was supposed to return data with lifetime `'1` but it is returning data with lifetime `'2`
+   |          |  |
+   |          |  has type `&'1u32`
```

Comment:
maybe missing whitespace between region and type(`u32`)?

Reply:
Oops! Thanks, never spotted these when looking through the changed tests, pushed a fix for that.